.changeset/eleven-olives-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Add support for an `--outDir` CLI flag to `astro build`
```

Comment:
```suggestion
Adds support for an `--outDir` CLI flag to `astro build`
```
New verb tense! Everyone get used to it!

Reply:
😱